src/error.rs

```diff
@@ -197,6 +197,9 @@ pub enum ErrorCode {
 	/// (because its IRI scheme matches a term definition and it has no IRI authority).
 	IriConfusedWithPrefix,
 
+	/// Unable to expand a key to a IRI or keyword in strict expansion mode
```

Comment:
Can you add a `.` at the end?

Reply:
I think it would also be preferable to mention that this error is not defined in the JSON-LD API specification.

Reply:
Update and rebased.

The spec mentions generating warnings; maybe a more standard way in the future could be to wrap return values with a type that includes a set of warnings. Then if a user wants the strict behavior they could check that there are no warnings.

Thanks for making `json-ld`.

Reply:
Yes I'm still looking for a nice way to handle warnings. I'm worried that putting them in return types might become annoying to manipulate for most users who don't care about them.